go.mod

```diff
@@ -3,24 +3,23 @@ module github.com/Agoric/agoric-sdk
 go 1.17
 
 require (
-	github.com/armon/go-metrics v0.3.9
-	github.com/cosmos/cosmos-sdk v0.45.1
-	github.com/cosmos/gaia/v6 v6.0.3
-	github.com/cosmos/ibc-go/v2 v2.0.3
+	github.com/armon/go-metrics v0.3.10
+	github.com/cosmos/cosmos-sdk v0.45.4
+	github.com/cosmos/gaia/v7 v7.0.0-00010101000000-000000000000
+	github.com/cosmos/ibc-go/v3 v3.0.0
```

Comment:
Reviewer - do we still need to use our own fork of ibc-go?

Reply:
Not at this time.  That's applicable only when we consider adopting Interchain Account host support.

[Our fork](https://github.com/agoric-labs/ibc-go/Agoric) implements ICA message type allowlist wildcards.  This is to more conveniently restrict Cosmos-layer messages if the product folks deem some of them a bad idea to enable on the Agoric chain.